pkgs/development/python-modules/flask-silk/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, flask
+}:
+
+buildPythonPackage rec {
+  pname = "Flask-Silk";
+  version = "0.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1gjzighx4f0w39sq9xvzr1kwb4y7yv9qrgzvli1p89gy16piz8l0";
+  };
+
+  propagatedBuildInputs = [
+    flask
+  ];
+
+  meta = with stdenv.lib; {
+    description = "Adds silk icons to your Flask application or module, or extension";
+    license = licenses.bsd2;
```

Comment:
https://github.com/sublee/flask-silk/blob/master/LICENSE looks like `bsd3` to me

Reply:
I was going after [this](https://en.wikipedia.org/wiki/BSD_licenses#3-clause_license_(%22BSD_License_2.0%22,_%22Revised_BSD_License%22,_%22New_BSD_License%22,_or_%22Modified_BSD_License%22), which doesn't list a `bsd3`. Why do you think its 3 and not 2?

Reply:
In
https://github.com/NixOS/nixpkgs/blob/92f0d31b949bcf018262252408b8a69376a6cb68/lib/licenses.nix#L92-L95
you can see that `bsd3` stands for the 3-clause version. Flask-silk's license is a variant of the 3-clause version.

Reply:
Ah I see.